ccd-definition/CaseEventToFields/CareSupervision/manageOrders/5_orderDetailsSection.json

```diff
@@ -688,5 +688,109 @@
+    "PageDisplayOrder": 6,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "orderTempQuestions.manageOrdersVaryOrExtendSupervisionOrder=\"YES\"",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "manageOrdersSupervisionOrderVariationHeading",
+    "PageFieldDisplayOrder": 33,
+    "DisplayContext": "READONLY",
+    "PageID": "OrderDetails",
+    "PageDisplayOrder": 6,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "manageOrdersSupervisionOrderType=\"VARIATION_OF_SUPERVISION_ORDER\" AND manageOrdersC35aOrderExists=\"Yes\"",
+    "ShowSummaryChangeOption": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "manageOrdersSupervisionOrderExtensionHeading",
+    "PageFieldDisplayOrder": 34,
+    "DisplayContext": "READONLY",
+    "PageID": "OrderDetails",
+    "PageDisplayOrder": 6,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "manageOrdersSupervisionOrderType=\"EXTENSION_OF_SUPERVISION_ORDER\" AND manageOrdersC35aOrderExists=\"Yes\"",
+    "ShowSummaryChangeOption": "N"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "manageOrdersSupervisionOrderType",
+    "PageFieldDisplayOrder": 35,
+    "DisplayContext": "MANDATORY",
+    "PageID": "OrderDetails",
+    "PageDisplayOrder": 6,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "manageOrdersC35aOrderExists=\"Yes\"",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "manageOrdersSupervisionOrderCourtDirection",
+    "PageFieldDisplayOrder": 36,
+    "DisplayContext": "MANDATORY",
+    "PageID": "OrderDetails",
+    "PageDisplayOrder": 6,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "manageOrdersC35aOrderExists=\"Yes\"",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "manageOrdersSupervisionOrderApprovalDate",
+    "PageFieldDisplayOrder": 37,
+    "DisplayContext": "MANDATORY",
+    "PageID": "OrderDetails",
+    "PageDisplayOrder": 6,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "manageOrdersC35aOrderExists=\"Yes\"",
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "manageOrders",
+    "CaseFieldID": "manageOrdersSupervisionOrderEndDate",
+    "PageFieldDisplayOrder": 38,
+    "DisplayContext": "MANDATORY",
+    "PageID": "OrderDetails",
+    "PageDisplayOrder": 6,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "manageOrdersC35aOrderExists=\"Yes\"",
+    "ShowSummaryChangeOption": "Y"
```

Comment:
Not sure what the best way to solve this is but this will conflict with my changes for display order in https://github.com/hmcts/fpl-ccd-configuration/pull/3500/files